flocker/node/script.py

```diff
@@ -154,13 +133,8 @@ class ZFSAgentScript(object):
     a Flocker cluster.
     """
     def main(self, reactor, options, volume_service):
-        try:
-            agent_config = options[u'agent-config']
-            configuration = yaml.safe_load(agent_config.getContent())
-        except IOError:
-            raise ConfigurationError(
-                "Configuration file does not exist at '{}'.".format(
-                    agent_config.path))
+        agent_config = options[u'agent-config']
```

Comment:
Is this https://github.com/ClusterHQ/flocker/pull/1403#discussion_r30401841 the suggestion you refer to?

With this change it'll once again be clear when the configuration file doesn't exist, is unreadable or unparsable etc....but with an ugly stack trace.

And you'll create a config loading function with proper error checking in FLOC-1791 (mentioned above)?